cpp/tests/data/ode-sir-js-compare.csv

```diff
@@ -0,0 +1,501 @@
+ # t S I R

```

Comment:
Although it is not 100kb, 40kb is still large for a file like this. You probably adapted well from the old file but we should stick to smaller files as the repo grows. I think comparing the first 20 steps is already sufficient to compare the results to some higher accuracy. If there is an imprecision of X after 100 iterations, there is an imprecision of Y after 10 iterations.